cl/alerts/tasks.py

```diff
@@ -265,7 +267,7 @@ def send_alert_and_webhook(
     DocketAlert.objects.filter(docket=d).update(date_last_hit=now())
 
     # Send the docket to webhook
-    send_docket_alert_webhooks.delay(d_pk, since, new_des)
+    send_docket_alert_webhooks.delay(new_des, da_recipients)
```

Comment:
Generally, we shouldn't be sending models as parameters to tasks, for a few reasons:

 - The objects can change while the item is in the queue
 - The objects are larger than the PKs
 - It only works when you have `CELERY_TASK_SERIALIZER` set to `pickle`, which we currently do, but someday we'll want to change to `json`, which is safer.

Can you refactor to send PKs, please?

Reply:
Got it, sure I'll refactor to use pks, thanks for the advice.

Reply:
Changed, now we use pks here instead of models.